src/SFA.DAS.EAS.Web/FileDownloads/Payments/ITransactionFormatter.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using SFA.DAS.EAS.Domain.Models.Transaction;
+using SFA.DAS.EAS.Web.ViewModels;
+
+namespace SFA.DAS.EAS.Web.FileDownloads.Payments
+{
+    public interface ITransactionFormatter
+    {
+        byte[] GetFileData(List<TransactionDownloadLine> dataPayments);
```

Comment:
If we define the List<T> as IEnumerable<T> would we make it more flexible? The concrete methods only do a foreach over the collections so don't need it to be a specific concrete type.

Reply:
So I copied this for David's work again. i'm make the edit now as you are correct it would be better to use that.